blogs/posts/2023-04-26_alternative_remotes.qmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Alternative remote repositories"
+author: "Tom Jemmett"
+date: "2023-04-26"
+categories: [git, tutorial]
+knitr:
+  opts_chunk:
+    fig.height: 2.5
+    echo: false
+    dev: png
+    dev.args:
+      bg: transparent
+---
+
+It's great when someone send's you a pull request on GitHub to fix bugs or add new features to your project, but you probably always want to check the other persons work in someway before merging that pull request.
+
+All of the steps below are intended to be entered via a terminal.
+
+Let's imagine that we have a GitHub account called example and a repository called test, and we use https rather than ssh.
+
+```
+$ git remote get-url origin
+# https://github.com/example/test.git
+```
+
+Now, let's say we have someone who has submitted a PR, and their username is friend. We can add a new remote for their fork with
```

Comment:
Can we write pull request (PR)? It's part of the style for writing that I'd like to introduce https://nhs-r-community.github.io/NHSR-way/community-handbook.html#explain-acronyms from NHS-R

Reply:
changed!